create-server-config.php

```diff
@@ -1,4 +1,15 @@
-ErrorDocument 404 /404.php
+<?php 
+/********************************************************************/
+//                       create-htaccess.php
+//                  Created by Yigit Kerem Oktay
+// This file generates a .htaccess file that contains all necessary 
+// code for it.
+// This is needed because some hosts do not either unzip hidden files
+// or neither GitHub puts that file inside the zips.
+/********************************************************************/
+if(strpos($_SERVER['SERVER_SOFTWARE'],'apache')!== false){
```

Comment:
Just noticed this - maybe replace with `stripos` so that it's case insensitive (just to be sure :slightly_smiling_face:  ) and we're good to go... :+1: Great work!

Reply:
Yes will need one more little thing